08-TabbedNavigation/src/PrismSample/ViewModels/MainPageViewModel.cs

```diff
@@ -1,11 +1,37 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
+﻿using Prism.Commands;
 using Prism.Mvvm;
+using Prism.Navigation;
+using PrismSample.Views;
+using Xamarin.Forms;
 
 namespace PrismSample.ViewModels
 {
     public class MainPageViewModel : BindableBase
     {
+        private readonly INavigationService _navigationService;
```

Comment:
readonly property/field declarations shouldn't be mixed... keep it consistent here `private INavigationService _navigationService { get; }`